### ISSUE TITLE
[DEB] Can't load the list-games.

### ISSUE THREAD

**Issue by Kazer67** [flavioislima/HeroicGamesLauncher#12]
Operating System: Pop!_OS 20.10 - 64 bits (Ubuntu based distro)
Gnome version: 3.38.1
Window system: X11
CPU: i7-4770K
GPU: GM204 [GeForce GTX 980]

- Xterm installed.
- .deb Heroic package installed

1) Launch Heroic from the command line
2) Heroic work as intended, so login process
![image](https://user-images.githubusercontent.com/45577220/103479778-4b7dbd00-4dd0-11eb-8474-3caed16eb746.png)
3) Use the internal browser to log to Epic account and copy the SID then paste it and click login
![image](https://user-images.githubusercontent.com/45577220/103479811-83850000-4dd0-11eb-8a06-3139b51360bc.png)
4)Wait for it to refresh the list, nothing in the log
![image](https://user-images.githubusercontent.com/45577220/103479822-97c8fd00-4dd0-11eb-88dd-c320fa95615d.png)
5) Then, when it's logged in, error in the terminal:
`Error occurred in handler for 'readFile': TypeError: Cannot read property 'url' of undefined`
    `at /opt/heroic/resources/app.asar/build/main.js:152:15`
    `at Array.map (<anonymous>)`
    `at /opt/heroic/resources/app.asar/build/main.js:145:10`
    `at async electron/js2c/browser_init.js:189:563`

![Capture d’écran du 2021-01-03 14-34-20](https://user-images.githubusercontent.com/45577220/103479881-fd1cee00-4dd0-11eb-95d3-593d09bd6e77.png)

6) Every time I reload or relaunch Heroic, same error in the terminal and black screen
![image](https://user-images.githubusercontent.com/45577220/103479905-20e03400-4dd1-11eb-82f7-a01641c6d71e.png)

**Comment by flavioislima** [comment]
This error is like some of the games has no Image URL. It is really weird because I have almost 90 games in my library and none of them have this issue. I'll add a fallback and post here a test .deb for you to try if it fixes.

**Comment by flavioislima** [comment]
I think it may be a DLC or something. I found some DLCs on my library that have no cover art, so might me it. 
I'll add a fallback cover like this one when no image is found:

![no-cover](https://user-images.githubusercontent.com/26871415/103480183-1fb00680-4dd3-11eb-9171-d8c4cc601fba.jpg)

**Comment by flavioislima** [comment]
@Kazer67 
I compiled a Deb file that should fix this. Download it here:
https://drive.google.com/file/d/1YNRDTIvVqCkk4Aj9JyKByFFX8O9qDhFd/view?usp=sharing

**Comment by Kazer67** [comment]
> @Kazer67
> I compiled a Deb file that should fix this. Download it here:
> https://drive.google.com/file/d/1YNRDTIvVqCkk4Aj9JyKByFFX8O9qDhFd/view?usp=sharing

This version work well. I started from scratch, cleaned both Legendary & Heroic in .config.

I launched Heroic and I logged in, it refresh the game list and once it's done, it say I'm logged with my user.

I then do a refresh (Reload with CTRL+R) and Heroic list all my games. Seem to be solved for now, I will try the rest to see if I find other issue.
